src/main/java/com/beerjournal/breweriana/item/ItemService.java

```diff
@@ -34,4 +34,21 @@ ItemDto addItem(String ownerId, ItemDto itemDto) {
         return itemDto;
     }
 
+    ItemDto deleteItem(String ownerId, String itemId) {
```

Comment:
I think you should also check whether user is authorized to delete an item from collection (i.e. user can only delete from his own collection) :P